ee/clickhouse/client.py

```diff
@@ -54,6 +54,7 @@ def make_ch_pool(**overrides) -> ChPool:
         connections_min=CLICKHOUSE_CONN_POOL_MIN,
         connections_max=CLICKHOUSE_CONN_POOL_MAX,
         settings={"mutations_sync": "1"} if TEST else {},
+        **overrides,
```

Comment:
Ah, missed this in the review, apologies!

For these overrides to work as expected, we'd need the arguments beforehand to be a dict as well, which are then overridden and unpacked.

Something like:

```
>>> a = {'1': 1, '2': 2}
>>> b = {'1': 3}
>>> {**a, **b}
{'1': 3, '2': 2}
```

ref: https://sentry.io/organizations/posthog/issues/2636682389/?referrer=slack

Reply:
https://github.com/PostHog/posthog/pull/5874